lib/core/src/lib/form/components/widgets/date/date.widget.ts

```diff
@@ -54,6 +54,10 @@ export class DateWidgetComponent extends WidgetComponent implements OnInit, OnDe
 
     minDate: Moment;
     maxDate: Moment;
+    startAt: Moment;
+
+    @Input()
+    value = null;
```

Comment:
should not we set some type?

Reply:
added "any" type to reflect the form field value